pkg/protocol/engine/accounts/accountsledger/snapshot.go

```diff
@@ -215,6 +215,8 @@ func (m *Manager) readSlotDiffs(reader io.ReadSeeker, slotDiffCount uint64) erro
 func (m *Manager) writeSlotDiffs(pWriter *utils.PositionedWriter, targetIndex iotago.SlotIndex) (slotDiffsCount uint64, err error) {
 	// write slot diffs until being able to reach targetIndex, where the exported tree is at
 	slotIndex := iotago.SlotIndex(1)
+
+	// TODO: shouldn't that be from last finalized slot?
```

Comment:
Not really: is the targetIndex that should be bounded depending on the case: unless you are switching engines, the targetIndex _shouldn't_ be above last finalized slot.